schemas/TemperatureCurve.yaml

```diff
@@ -21,6 +23,14 @@ properties:
         feedTemperature: 75.0
       - outdoorTemperature: 0.0
         feedTemperature: 57.0
```

Comment:
Add a ` required` for `outdoorTemperature` and `feedTemperature` in `breakpoints.items` as well.
```
required:
  - outdoorTemperature
  - feedTemperature
```